ltac/tacinterp.ml

```diff
@@ -1429,7 +1427,8 @@ and tactic_of_value ist vle =
       let tac = name_if_glob appl (eval_tactic ist t) in
       Profile_ltac.do_profile "tactic_of_value" trace (catch_error_tac trace tac)
   | VFun (_, _, _,vars,_) ->
-    Tacticals.New.tclZEROMSG (str "A fully applied tactic is expected: missing argument(s) " ++ arg_names vars)
+    let plural = if List.length vars > 1 then Pp.str "s" else Pp.str "" in
```

Comment:
Please add a `pluralize : string -> 'a list -> std_ppcmds` function to `Pp` and use it instead.